op-guide/ansible-deployment.md

```diff
@@ -207,6 +207,13 @@ pd_servers
         ansible-playbook bootstrap.yml -k
         ```
 
+        或者：
+        如果 Ansible 使用普通用户远程连接需要密码, 可以添加 --extra-vars 参数，执行其他 playbook 同理：
+
+        ```
+        ansible-playbook bootstrap.yml --extra-vars "ansible_sudo_pass=tidbpasswd"
```

Comment:
not the best practice. This will leave your sudo password in shell history.

Better add link to http://docs.ansible.com/ansible/latest/playbooks_vault.html

Reply:
Let me see，thx